data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/CsaProfileCrac.java

```diff
@@ -87,15 +87,51 @@ public PropertyBags getShuntCompensatorModifications() {
     }
 
     public PropertyBags getTapPositionAction() {
-        return this.queryTripleStore(CsaProfileConstants.TAP_POSITION_ACTION);
+        return this.queryTripleStore(CsaProfileConstants.TAP_POSITION_ACTION, tripleStoreCsaProfileCrac.contextNames());
```

Comment:
We need to investigate why the `tripleStoreCsaProfileCrac.contextNames()` came back because they were removed in a previous PR